File: ticketProject7-CrossPlatform/TS-crossp/networking/request/LoginRequest.cs
using System;
namespace networking.protocol
{

	using SellerDTO = networking.dto.SellerDTO;

	[Serializable]
	public class LoginRequest : Request
	{
		private SellerDTO user;

		public LoginRequest(SellerDTO user)
		{
			this.user = user;
		}

		public virtual SellerDTO Seller
		{
			get
			{
				return user;
			}
		}
	}

}

File: ticketProject6-RMI/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/LoginRequest.java
package network.objectprotocol;

import network.dto.SellerDTO;

public class LoginRequest implements Request {
    private SellerDTO seller;

    public LoginRequest(SellerDTO seller) {
        this.seller = seller;
    }

    public SellerDTO getSeller() {
        return seller;
    }
}
